doc/pandas.rst

```diff
@@ -125,3 +125,67 @@ xarray objects do not yet support hierarchical indexes, so if your data has
 a hierarchical index, you will either need to unstack it first or use the
 :py:meth:`~xarray.DataArray.from_series` or
 :py:meth:`~xarray.Dataset.from_dataframe` constructors described above.
+
+
+Transitioning from pandas' Panels to xarray
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+[Preamble about deprecating Panel to xarray etc]
+
+xarray has most of Panel's features, a more explicit API (particularly around
+indexing), and the ability to scale to >3 dimensions with the same interface.
+
+As discussed in the xarray docs, there are two primary data structures in xarray:
+DataArray and Dataset. You can imagine a DataArray as a n-dimensional pandas
+Series (i.e. a single typed array), and a Dataset as the DataFrame-equivalent
+(i.e. a dict of aligned DataArray-s).
+
+So you can represent a Panel, in two ways:
+- A 3-dimenional DataArray
+- A Dataset containing a number of 2-dimensional DataArray-s
+
+.. ipython:: python
+    panel=pd.Panel( pd.np.random.rand(2,3,4), items=list('ab'), major_axis=list('mno'),
+        minor_axis=pd.date_range(start='2000', periods=4, name='date'))
+
+    panel
+
+
+As a DataArray:
+
+
+.. ipython:: python
+
+    xr.DataArray(panel)
```

Comment:
Note that recent versions of pandas have a `.to_xarray()`, which does the exact same thing.